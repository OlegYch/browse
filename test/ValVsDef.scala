class ValVsDef {
  val x = 1
  val y = 1
  val lol = x + y
  def rofl = x + lol
}

