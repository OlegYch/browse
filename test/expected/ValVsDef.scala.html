<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8" ></meta>
        <title>ValVsDef.scala</title>
        <script type="text/javascript" src="jquery-all.js"></script>
        <script type="text/javascript" src="linked.js"></script>
        <link rel="stylesheet" type="text/css" href="style.css" title="Style"></link>
    </head>
    <body>
        <pre>
<span class="keyword">class</span> <a title="class ValVsDef extends AnyRef" id=";ValVsDef">ValVsDef</a> <a href="#;ValVsDef" title="ValVsDef" class="delimiter">{</a>
  <span class="keyword">val</span> <a title="Int" id=";ValVsDef.x">x</a> = <span title="Int(1)" class="int">1</span>
  <span class="keyword">val</span> <a title="Int" id=";ValVsDef.y">y</a> = <span title="Int(1)" class="int">1</span>
  <span class="keyword">val</span> <a title="Int" id=";ValVsDef.lol">lol</a> = <a href="#;ValVsDef.x" title="=&gt; Int">x</a> <span title="(x: Int)Int">+</span> <a href="#;ValVsDef.y" title="=&gt; Int">y</a>
  <span class="keyword">def</span> <a title="=&gt; Int" id=";ValVsDef.rofl">rofl</a> = <a href="#;ValVsDef.x" title="=&gt; Int">x</a> <span title="(x: Int)Int">+</span> <a href="#;ValVsDef.lol" title="=&gt; Int">lol</a>
<span class="delimiter">}</span>

        </pre>
    </body>
</html>
